src/test/java/org/prebid/server/handler/CookieSyncHandlerTest.java

```diff
@@ -245,7 +246,7 @@ public void shouldRespondWithBadRequestStatusIfGdprConsentIsInvalid() {
         // given
         given(routingContext.getBody())
                 .willReturn(givenRequestBody(CookieSyncRequest.builder()
-                        .bidders(emptyList())
+                        .bidders(Collections.singletonList(RUBICON))
```

Comment:
In tests we'd rather prefer to use static method imports to rid off `Collections.`, etc.